Change file extension of exported compose file to ".yml"

Double-A-92 opened the pull request:
I ran into a tricky issue when trying to create my own AppStore with my custom apps. 

I exported their compose files from the settings view, and that created a "appname.**yaml**" file. So far, so good.
However the AppStore loader only accepts ".yml" files (without the a), which was an easy thing to miss and lead me to long debugging...

Maybe this little change will prevent other people from having the same issue.

zhanghengxin commented:
we should fix filter and should not exported extension.

zhanghengxin commented:
![image](https://github.com/IceWhaleTech/CasaOS-UI/assets/24197448/5d62f843-0b6d-4dcc-b492-27337fabc441)
Is this the store loader you were talking about?

zhanghengxin commented:
duplicate https://github.com/IceWhaleTech/CasaOS-AppManagement/pull/123

Double-A-92 commented:
I have created my own store ( https://github.com/Double-A-92/CasaOS-AppStore ) and it didn't work with .yaml files. When I added the store, it showed 0 new apps.

It is now fixed with https://github.com/IceWhaleTech/CasaOS-AppManagement/pull/123 .

I still wanted to make "yml" the standard everywhere, because it is mainly used internally in CasaOS-AppManagement. 
But it's not very important anymore.

zhanghengxin commented:
I think that we should look forward and follow industry standards.

Of course. I strongly believe that compatibility is always the right choice.
